ChopChop/ViewModels/CreateUserProfileViewModel.swift

```diff
@@ -19,17 +19,19 @@ final class CreateUserProfileViewModel: ObservableObject {
             return
         }
 
-        guard let newUser = try? User(name: name) else {
+        // Its not possible to create a user with just aname
+        guard !name.isEmpty else {
             errorMessage = "Empty name not accepted"
             return
         }
 
-        guard let userId = try? storageManager.createUser(user: newUser) else {
-            assertionFailure()
-            return
+        let userId = try? storageManager.addUser(name: name) { err in
+            if err != nil {
+                assertionFailure()
+                return
+            }
         }
 
-        UserDefaults.standard.set(userId, forKey: "userId")
```

Comment:
Why was this removed?

Reply:
Realised that its already set when i update userId of UserSettings so no longer necessary